manage/upgrading/version_specific_migration/upgrade_to_python3.rst

```diff
@@ -107,13 +115,13 @@ Then the source of the add-on package will be checked out into the ``src`` folde
 
 Now everything is prepared to work on the migration of the package.
 
-However, it is a good idea to now try
+However, for small or simple packages it is a good idea to now try
```

Comment:
Please follow our style guide!
https://docs.plone.org/about/contributing/word_choice.html

Thanks!

Reply:
Fixed in 9f40573.